tests/network/vmi_networking.go

```diff
@@ -259,36 +259,36 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			By("checking the device vendor in /sys/class")
 			// Create a machine with e1000 interface model
 			// Use alpine because cirros dhcp client starts prematurely before link is ready
-			masqIface := libvmi.InterfaceDeviceWithMasqueradeBinding()
-			masqIface.Model = "e1000"
-			masqIface.PciAddress = "0000:02:01.0"
+			e1000ModelIface := libvmi.InterfaceDeviceWithMasqueradeBinding()
+			e1000ModelIface.Model = "e1000"
+			e1000ModelIface.PciAddress = "0000:02:01.0"
 
-			bridgeIface := libvmi.InterfaceDeviceWithBridgeBinding(secondaryNetName)
-			bridgeIface.PciAddress = "0000:03:00.0"
+			defaultModelIface := libvmi.InterfaceDeviceWithBridgeBinding(secondaryNetName)
+			defaultModelIface.PciAddress = "0000:03:00.0"
 
 			bridgeNetwork := libvmi.MultusNetwork(secondaryNetName, nadName)
-			e1000VMI := libvmifact.NewAlpine(
-				libvmi.WithInterface(masqIface),
+			multiIfaceVMI := libvmifact.NewAlpine(
+				libvmi.WithInterface(e1000ModelIface),
 				libvmi.WithNetwork(v1.DefaultPodNetwork()),
-				libvmi.WithInterface(bridgeIface),
+				libvmi.WithInterface(defaultModelIface),
```

Comment:
These were introduced in the previous commit, I think they could have their final name on introduction.